src/pudl_archiver/archivers/classes.py

```diff
@@ -311,7 +338,7 @@ async def get_hyperlinks(
         if not hyperlinks:
             self.logger.warning(
                 f"The archiver couldn't find any hyperlinks{('that match: ' + filter_pattern.pattern) if filter_pattern else ''}."
```

Comment:
```suggestion
                f"The archiver couldn't find any hyperlinks {('that match: ' + filter_pattern.pattern) if filter_pattern else ''}."
```
Non-blocking: while we're here we can fix a spacing issue.